Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -141,6 +141,52 @@ public void Revoke(IdentityAddress activeIdentity, DeviceId activeDevice)
         AuditLog.Add(auditLogEntry);
     }
 
+    public void RevokeReactivation(IdentityAddress activeIdentity, DeviceId activeDevice)
+    {
+        EnsureOpenReactivationRequestExists(activeIdentity);
+
+        var auditLogEntry = new RelationshipAuditLogEntry(
+            RelationshipAuditLogEntryReason.RevocationOfReactivation,
+            RelationshipStatus.Terminated,
+            RelationshipStatus.Terminated,
+            activeIdentity,
+            activeDevice
+        );
+        AuditLog.Add(auditLogEntry);
+    }
+
+    private void EnsureOpenReactivationRequestExists(IdentityAddress target)
```

Comment:
I would rather call it "activeIdentity" than "target". "target" sounds like the target of the reactivation request.

Reply:
Makes sense.

Changes made.